cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -44,6 +44,8 @@ final class DataprocConf {
   static final String CLOUD_PLATFORM_SCOPE = "https://www.googleapis.com/auth/cloud-platform";
 
   static final String TOKEN_ENDPOINT_KEY = "token.endpoint";
+  static final String COMPUTE_CREDENTIALS_MAX_RETRIES = "compute.credentials.max.retries";
```

Comment:
NIT : 
Please these to below `COMPUTE_HTTP_REQUEST_READ_TIMEOUT_DEFAULT` , around line 110 . 

So the Key and Defaults are placed together for better readability.